src/components/Comments.jsx

```diff
@@ -3,21 +3,21 @@ import { useState, useEffect } from "react";
 import { getComments } from "../api";
 import CommentAdder from "./CommentAdder";
 
-function Comments() {
+function Comments({ article_id }) {
   const [comments, setComments] = useState([]);
 
   useEffect(() => {
-    getComments().then(({ comments }) => {
-      setComments(comments);
+    getComments(article_id).then((res) => {
+      setComments(res.data.comments);
     });
-  }, []);
+  }, [article_id]);
 
   return (
     <main>
       {comments.map((comment) => {
         return <CommentCard key={comment.comment_id} {...comment} />;

```

Comment:
There is no need to spread the comment, we are only viewing them. We don't have any ability to mutate a comment from here.